src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/adjudications/AdjudicationTransformationTest.kt

```diff
@@ -589,6 +593,77 @@ class AdjudicationTransformationTest {
       )
     }
   }
+
+  @Nested
+  inner class Hearings {
+    @Test
+    fun `will copy core hearing details`() {
+      val nomisAdjudication = nomisAdjudicationCharge(
+        hearings = listOf(
+          Hearing(
+            hearingId = 54321,
+            hearingDate = LocalDate.parse("2021-01-01"),
+            hearingTime = "12:00:00",
+            type = CodeDescription(code = "GOV_ADULT", description = "Governor's Hearing Adult"),
+            hearingResults = emptyList(),
+            scheduleDate = LocalDate.parse("2020-12-31"),
+            scheduleTime = "11:00:00",
+            comment = "Some comment",
+            hearingStaff = Staff(
+              username = "A.JUDGE",
+              staffId = 123,
+              firstName = "A",
+              lastName = "JUDGE",
+            ),
+            internalLocation = InternalLocation(321, "A-1-1", "MDI-A-1-1"),
+            eventStatus = CodeDescription(code = "SCH", description = "Scheduled"),
+          ),
+        ),
+      )
+      val dpsAdjudication = nomisAdjudication.toAdjudication()
+
+      assertThat(dpsAdjudication.hearings).containsExactly(
+        MigrateHearing(
+          oicHearingId = 54321,
+          oicHearingType = GOV_ADULT,
+          hearingDateTime = "2021-01-01T12:00:00",
+          adjudicator = "A.JUDGE",
+          commentText = "Some comment",
+          locationId = 321,
+          hearingResult = null,
+        ),
+      )
+    }
+
+    @Test
+    fun `will copy scheduled hearing details`() {
+      val nomisAdjudication = nomisAdjudicationCharge(
+        hearings = listOf(
+          Hearing(
+            hearingId = 54321,
+            hearingDate = LocalDate.parse("2021-01-01"),
+            hearingTime = "12:00:00",
+            hearingResults = emptyList(),
+            internalLocation = InternalLocation(321, "A-1-1", "MDI-A-1-1"),
+            eventStatus = CodeDescription(code = "SCH", description = "Scheduled"),
+          ),
+        ),
+      )
+      val dpsAdjudication = nomisAdjudication.toAdjudication()
+
+      assertThat(dpsAdjudication.hearings).containsExactly(
+        MigrateHearing(
+          oicHearingId = 54321,
+          oicHearingType = GOV, // TODO - we always havce a NOMIS type so default to this until we have a decision
+          hearingDateTime = "2021-01-01T12:00:00",
+          adjudicator = null,
+          commentText = null,
+          locationId = 321,
+          hearingResult = null,
```

Comment:
Do you need a test for when this is not null?

Reply:
100% that is the next phase - there is a whole thing about results not touched yet